docs/dev/adding_instruments.rst

```diff
@@ -531,15 +580,56 @@ The same can be also achieved by the `preprocess_reply` keyword argument to :fun
 
     Extreme5000.capacity = Instrument.measurement(
         ":CAP?",
-        """ A measurement returning a capacity in nF in the format '<cap> nF'
-        """,
+        """Measure the capacity in nF (float).""",
         preprocess_reply=lambda v: v.replace('nF', '')
         # notice how we don't need to cast to float anymore
     )
 
+Tweaking command strings
+************************
+If you need to tweak
+
+* the :code:`set_command` string immediately before the value to set is inserted via string formatting (:code:`%g` etc.), or
+* the :code:`get_command` string before sending it to the device,
+
+use the :code:`command_process` parameter of :meth:`~pymeasure.instruments.Instrument.control`.
+
+Note that there is only parameter for both setting and getting, so the utility of this is probably limited.
+Note also that for adding e.g. channel identifiers, there are other, more preferable methods.
```

Comment:
```suggestion
Note, that there is only one parameter for both setting and getting, so the utility of this is probably limited.
Note also, that for adding e.g. channel identifiers, there are other, more preferable methods.
```

Reply:
No comma here, either, "note that the sky is blue" is one phrase. I added the "one", though. :-P

Reply:
You're right.